src/Adapters/Phpunit/Iteration.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\Collision\Adapters\Phpunit;
+
+/**
+ * @internal
+ */
+class Iteration
+{
+    /**
+     * The current iteration of a repeated test.
+     *
+     * @readonly
+     *
+     * @var int|null
+     */
+    public $iteration = null;
```

Comment:
This is based on the Laravel Sequence class which behaves pretty much the same.
https://github.com/laravel/framework/blob/8.x/src/Illuminate/Database/Eloquent/Factories/Sequence.php

```suggestion
    public $index = 0;
```

Reply:
Closed as index is zero-based normally